versions.json

```diff
@@ -350,47 +406,55 @@
     "react": {
         "react-components": {
             "jsVersion": "24.4.0-alpha13",
-            "npmName": "@vaadin/react-components"
+            "npmName": "@vaadin/react-components",
```

Comment:
The overall "react-components" json can be moved to "vaadin" I believe to be consistent with the overall structure. But this might be a breaking change for platform, so let's make it in another PR at some point. Not a blocker at all.